### ISSUE TITLE
Quokka failing to start

### ISSUE THREAD

**Issue by timcosta** [wallabyjs/quokka#10]
### Issue description or question
Quokka failing to start
​​​​

### Quokka.js file

```javascript
import BaseModel from './models/base_model';

const a = new BaseModel('test1');
const b = new BaseModel('test2');
console.log(b._knex === a._knex);
```

In my package.json:
```
"quokka": {
    "babel": true
  }
```

### Quokka.js Console Output

```
​​​​Quokka #1 (node: v6.8.1, babel: v6.24.0)​​​​
​​Error: Configured file paths must be relative: /quokka.js​​
​​    at EventEmitter._createFileMetadata (/Users/tcosta/.vscode-insiders/extensions/WallabyJs.quokka-vscode-1.0.7/dist/wallaby/server.js:17:28452)​​
​​    at EventEmitter._getOrCreateFileMetadata (/Users/tcosta/.vscode-insiders/extensions/WallabyJs.quokka-vscode-1.0.7/dist/wallaby/server.js:17:28125)​​
​​    at /Users/tcosta/.vscode-insiders/extensions/WallabyJs.quokka-vscode-1.0.7/dist/wallaby/server.js:17:10162​​
​​    at baseFor (/Users/tcosta/.vscode-insiders/extensions/WallabyJs.quokka-vscode-1.0.7/dist/wallaby/node_modules/lodash/index.js:1971:13)​​
​​    at baseForOwn (/Users/tcosta/.vscode-insiders/extensions/WallabyJs.quokka-vscode-1.0.7/dist/wallaby/node_modules/lodash/index.js:2025:14)​​
​​    at baseEach (/Users/tcosta/.vscode-insiders/extensions/WallabyJs.quokka-vscode-1.0.7/dist/wallaby/node_modules/lodash/index.js:1814:16)​​
​​    at Function.forEach (/Users/tcosta/.vscode-insiders/extensions/WallabyJs.quokka-vscode-1.0.7/dist/wallaby/node_modules/lodash/index.js:5701:11)​​
​​    at EventEmitter._processChangedQueue (/Users/tcosta/.vscode-insiders/extensions/WallabyJs.quokka-vscode-1.0.7/dist/wallaby/server.js:17:9290)​​
​​    at Timeout._onTimeout (/Users/tcosta/.vscode-insiders/extensions/WallabyJs.quokka-vscode-1.0.7/dist/wallaby/server.js:17:4091)​​
​​    at ontimeout (timers.js:365:14)​​
​​    at tryOnTimeout (timers.js:237:5)​​
​​    at Timer.listOnTimeout (timers.js:207:5)​​
​​​
```

### Code editor version

Visual Studio Code v1.11.0-insider

### OS name and version
OSX 10.12.3

**Comment by nixterrimus** [comment]
I'm seeing this too:

```
​​​​Quokka #1 (node: v7.5.0, TypeScript: v2.1.6)​​​​
​​Error: Configured file paths must be relative: /quokka.ts​​
​​    at Project._createFileMetadata (/Users/nickrowe/.vscode/extensions/WallabyJs.quokka-vscode-1.0.7/dist/wallaby/server.js:17:28452)​​
​​    at Project._getOrCreateFileMetadata (/Users/nickrowe/.vscode/extensions/WallabyJs.quokka-vscode-1.0.7/dist/wallaby/server.js:17:28125)​​
​​    at /Users/nickrowe/.vscode/extensions/WallabyJs.quokka-vscode-1.0.7/dist/wallaby/server.js:17:10162​​
​​    at baseFor (/Users/nickrowe/.vscode/extensions/WallabyJs.quokka-vscode-1.0.7/dist/wallaby/node_modules/lodash/index.js:1971:13)​​
​​    at baseForOwn (/Users/nickrowe/.vscode/extensions/WallabyJs.quokka-vscode-1.0.7/dist/wallaby/node_modules/lodash/index.js:2025:14)​​
​​    at baseEach (/Users/nickrowe/.vscode/extensions/WallabyJs.quokka-vscode-1.0.7/dist/wallaby/node_modules/lodash/index.js:1814:16)​​
​​    at Function.forEach (/Users/nickrowe/.vscode/extensions/WallabyJs.quokka-vscode-1.0.7/dist/wallaby/node_modules/lodash/index.js:5701:11)​​
​​    at Project._processChangedQueue (/Users/nickrowe/.vscode/extensions/WallabyJs.quokka-vscode-1.0.7/dist/wallaby/server.js:17:9290)​​
​​    at Timeout._onTimeout (/Users/nickrowe/.vscode/extensions/WallabyJs.quokka-vscode-1.0.7/dist/wallaby/server.js:17:4091)​​
​​    at ontimeout (timers.js:365:14)​​
​​    at tryOnTimeout (timers.js:237:5)​​
​​    at Timer.listOnTimeout (timers.js:207:5)​​
​​​​```

VSCode: Version 1.10.2 (1.10.2)
OS X: 10.12.2 (16C67)

**Comment by hinok** [comment]
Similiar, I'm just trying to import a module and use it.

```
​​​​Quokka #1 (node: v6.9.1, babel: v6.23.1)​​​​
​​​​
​​Unexpected token import​​
  at ​​​Object.exports.runInThisContext​​​ ​vm.js:76​
```

I set `{"babel": true}` in `~/.quokka/config.json`, restarted vs code.

```
VS Code 1.10.2
mac OS 10.12.3
```

**Comment by ArtemGovorov** [comment]
The issue is fixed in v1.0.8, the version is published.

@hinok Your issue seem to be completely different from the originally reported. try this sample to see if it's working for you: https://github.com/ArtemGovorov/quokka-5, clone the repo, and run this quokka:

```javascript
import value from './value.js'

console.log(value)
```

**Comment by hinok** [comment]
@ArtemGovorov I got it! Your repo works and it helped me to find the root of the problem.

My error appeared because I was trying to run Quokka in a webpack project where [modules are disabled in babelrc](https://webpack.js.org/guides/migrating/#mixing-es2015-with-amd-and-commonjs), like

.babelrc

```js
{
  "presets": [
    ["es2015", {"modules":false}] // if you set modules to false, you will get the same error
  , "stage-2"],
  "plugins": ["transform-runtime"],
  "comments": false
}
```
